dist/static/site.css

```diff
@@ -0,0 +1,366 @@
+  width: 100%;
+  padding: 10px 15px;
+  margin: 10px 0;
+}
+
+button {
+  background: transparent;
+  color: #888;
+  font-size: 16px;
+  font-weight: 600;
+  margin: 8px 0;
+  padding: 10px;
+  border: none;
+  cursor: pointer;
+  transition: 120ms ease-in-out;
+  transition-property: background-color padding color;
+}
+
+button:after {
+  content: "\2192";
+  transition: 120ms ease-in-out;
+  margin-left: 6px;
+}
+
+button:hover:after {
+  margin-left: 10px;
+}
+
+button:hover {
+  background: #a50bca;
+  color: #fff;
+  padding: 10px 20px;
+}
+
+main {
+  margin: 0 16vw;
+}
+
+.song {
+  display: flex;
+  height: 90px;
+  background-color: rgba(0, 0, 0, 0.5);
+  border-radius: 5px;
+  overflow: hidden;
+  align-items: center;
+}
+
+.name {
+  text-shadow: 2px 2px 4px #000000;
+  font-family: "Righteous", monospace;
+}
+
+#nowplaying {
+  cursor: default;
+}
+
+#nowplaying .song {
+  width: 100%;
+  overflow: hidden;
+  border: none;
+  background: #111;
+  height: 90px;
+  align-items: center;
+}
+
+#nowplaying .song > .songtitle {
+  font-size: 1.2em;
+  font-family: "Righteous", monospace;
+  max-width: none;
+}
+
+#nowplaying button {
+  align-self: center;
+}
+
+@keyframes push-desc-out {
+  from {
+    transform: translateX(0px);
+  }
+  100% {
+    transform: translateX(30px);
+  }
+}
+
+#currentsong {
+  margin-top: 10px;
+  margin-bottom: 10px;
+}
+
+#currentplayer-descr {
+  font-family: "Righteous", monospace;
+  animation: 250ms pop-in ease, 400ms push-desc-out ease 250ms;
+  padding: 10px 0;
+  position: relative;
+  transform: translateX(30px);
+  text-shadow: 2px 2px 4px #000000;
+}
+
+@keyframes rotating-disc {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+@keyframes fade-in {
+  from,
+  40% {
+    opacity: 0;
+  }
+  to {
+    opacity: 1;
+  }
+}
+
+/* The disc animation */
+#currentplayer-disc {
+  position: absolute;
+  left: -30px;
+  animation: rotating-disc 3s linear infinite, fade-in 600ms ease;
+}
+
+.song img {
+  width: auto;
+  max-width: 120px;
+  height: 90px;
+  min-width: 90px;
+  opacity: 0.95;
+  float: left;
+}
+
+.song img:hover {
+  opacity: 1;
+}
+
+.songtitle {
+  height: 100%;
+  display: flex;
+  flex: 1;
+  align-self: center;
+  align-items: center;
+  margin: 10px;
+  max-width: 240px;
+}
+
+.songList > * {
+  margin: 5px 0;
+}
+
+/* Tab Menu */
+
+#tab-menu {
+  background: #111;
+  margin-top: 10px;
+  border-radius: 5px;
+}
+
+#tab-menu nav {
+  display: flex;
+  border-radius: 5px;
+}
+
+.menu-item {
+  display: inline-block;
+  font-weight: 600;
+  padding: 10px 20px;
+  cursor: pointer;
+  opacity: 0.6;
+  border-bottom: 4px solid transparent;
+  transition: 150ms ease;
+}
+
+.menu-item:hover,
+.menu-item.active {
+  opacity: 1;
+}
+
+.menu-item.active {
+  border-color: #a50bca;
+}
+
+#tab-queuesong,
+#tab-settings {
+  padding: 10px 20px;
+}
+
+#tab-queuesong label {
+  display: inline-block;
+  margin-top: 10px;
+}
+
+#input-songurl {
```

Comment:
Is it possible to add a different appearance for when the input box is "disabled"? This was something I forgot to add, but I think it will make it more clear to the user that it's not accepting input.